packages/terra-worklist-data-grid/src/subcomponents/Row.module.scss

```diff
@@ -7,6 +7,10 @@
     background-color: var(--terra-worklist-data-grid-row-striped-background-color, #f0f0f0);
   }
 
+  .worklist-data-grid-row:nth-child(odd) {
+    background-color: var(--terra-worklist-data-grid-row-striped-background-color, white);
+  }
```

Comment:
This is incorrect.  You can't use the same color for both even and odd rows.  You would need different variables.